imgui_widgets.cpp

```diff
@@ -720,6 +720,61 @@ bool ImGui::Button(const char* label, const ImVec2& size_arg)
     return ButtonEx(label, size_arg, ImGuiButtonFlags_None);
 }
 
+bool ImGui::BeginButtonDropDown(const char* label, ImVec2 buttonSize)
+{
+    ImGui::SameLine(0.f, 0.f);
```

Comment:
I don't think this should be here, calling `SameLine` is the user's  responsibility.